docs/development/README.adoc

```diff
@@ -1,10 +1,13 @@
-:toc: macro
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
 
 = T developer documentation
 
-toc::[]
-
-== <<local-t-network.adoc,Setting up local T network>>
+== xref:development/local-t-network.adoc[Setting up local T network]
```

Comment:
Sorry Kuba, I made a mistake in my suggestion. There should be no `development/` in the link:
```suggestion
== xref:local-t-network.adoc[Setting up local T network]
```